### ISSUE TITLE
Text rendering issue when maximizing window

### ISSUE THREAD

**Issue by yongjian-hu** [zim-desktop-wiki/zim-desktop-wiki#2082]
**Describe the bug**
Text rendering issue when maximizing the window. 

**To Reproduce**
1. Open any notebook or help page.
2. Increase window size
3. Text rendered incorrectly.

**Expected behavior**
Text rendered correctly when window size increased

**Screenshots**
1. Small window - text rendered correctly
<img width="789" alt="Screen Shot 2022-06-10 at 14 14 35" src="https://user-images.githubusercontent.com/32093451/173072458-6d56bc03-8514-4c7c-ba8a-b16387f4ffda.png">
<img width="864" alt="Screen Shot 2022-06-10 at 14 14 51" src="https://user-images.githubusercontent.com/32093451/173072471-9f3f41ba-0aa1-43a0-ae90-71619d13d5f3.png">
2. Increase window size - text rendered incorrectly
<img width="1072" alt="Screen Shot 2022-06-10 at 14 17 03" src="https://user-images.githubusercontent.com/32093451/173072646-a21721a3-509b-4440-938a-91cd856b2269.png">
<img width="1075" alt="Screen Shot 2022-06-10 at 14 17 01" src="https://user-images.githubusercontent.com/32093451/173072657-60a47b2b-09d8-4f3b-8ffb-5de7f147794b.png">
3. Select the text - display text correctly
<img width="1072" alt="Screen Shot 2022-06-10 at 14 18 19" src="https://user-images.githubusercontent.com/32093451/173073224-d827a433-70cf-4079-8dec-d27f2273274f.png">

**Environment (please complete the following information):**
 - Version [Zim-0.74.3_121_x86_64]
 - OS: macOS 11.6.5
 - Language: English US

**Additional context**
Similar issue #2042

**Comment by dehesselle** [comment]
To make reproducibility easy, let's stick to the "about" page from the help as our test case. I can reproduce the issue.

Can you confirm that the issue does not surface with [v0.74.3_80](https://gitlab.com/dehesselle/zim_macos/-/releases/v0.74.3_80)?

**Comment by yongjian-hu** [comment]
> To make reproducibility easy, let's stick to the "about" page from the help as our test case. I can reproduce the issue.
> 
> Can you confirm that the issue does not surface with [v0.74.3_80](https://gitlab.com/dehesselle/zim_macos/-/releases/v0.74.3_80)?

Hi, I just reinstalled and tested the version v0.74.3_80. The issue does not appear on this version so far.

**Comment by dehesselle** [comment]
Ok, that means the issue is caused by a dependency update that has happened between these versions. That might take some time to figure this out (lots of rebuilds and gradually rolling back updates) but at least I know now where to look.  
I'll ping you once I have a new build ready to confirm the fix.

**Comment by purejava** [comment]
This also happens on text that contains quotation marks, independently from maximizing the windows, e.g.:
```markdown
===== mailcow frontend konfigurieren =====
Wenn Mailboxen angelegt wurden, als die User an der ''mailcow UI'' anmelden und die Einstellungen der User-Accounts checken. Dort muss idR nur unter den Mailbox-Einstellungen konfiguriert werden, dass getaggte Mails in Unterordner verschoben werden.
```
<img width="565" alt="Bildschirmfoto 2022-06-11 um 14 47 44" src="https://user-images.githubusercontent.com/1822238/173188623-29b086e6-0f03-419d-bc60-7f4383b9731e.png">

**Comment by dehesselle** [comment]
Fixed with release [v0.74.3_153](https://gitlab.com/dehesselle/zim_macos/-/releases/v0.74.3_153).
